### PR TITLE
Add cf-modals

### PR DESCRIPTION
**Pull request by anselmbradford**
Adds https://github.com/cfpb/capital-framework/pull/583/ to prerelease-ver-6 branch

### CODE REVIEW

File: src/cf-modals/src/Modal.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
+/* ==========================================================================
+   Modal Organism
+   ========================================================================== */
+
+const UNDEFINED = require( 'cf-atomic-component/src/utilities/config' ).UNDEFINED;
```
Comment (anselmbradford) [inline src/cf-modals/src/Modal.js]: I don't see why [this is fine](https://github.com/cfpb/capital-framework/blob/canary/src/cf-expandables/src/Expandable.js#L18), but it's not fine here.

**Comment by anselmbradford** [comment]
@jimmynotjim did we figure out on Friday that the issue here in Travis is that cf-modals is not on npm yet?

**Comment by jimmynotjim** [comment]
@anselmbradford No, I don't think that's the case. I've got a bit of time so I'll see if I can track this down locally.

**Comment by jimmynotjim** [comment]
Figured it out, you used `atomic-component` in [package.json](https://github.com/cfpb/capital-framework/blob/canary/src/cf-expandables/package.json#L13) instead of `cf-atomic-component`. Update that and it'll build fine.

**Comment by jimmynotjim** [comment]
Since this was already approved and is going into the prerelease branch I'm going to merge it. If anyone has comments we can address them in a new PR.

**Pull request merged**